Sources/SwiftSyntax/Trivia.swift

```diff
@@ -15,8 +15,14 @@ public enum TriviaPosition {
   case trailing
 }
 
-/// A collection of leading or trailing trivia. This is the main data structure
-/// for thinking about trivia.
+/// Trivia represent pieces of the source code that are not relevant to represent
+/// its semantic structure.
```

Comment:
Might be worth pointing out here why we keep trivia at all, ie. to quote from the parser design principles:
> so that a syntax tree can be rendered back into text that is byte-for-byte identical to the original source